docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,248 @@
+= 5.4.0 Release Notes
+
+== New Features
+
+* **xref:???.adoc[???]**: ???
+
+== Breaking Changes
+
+* Moved the MongoDB connector to the extensions module, i.e., its classes and related dependencies relocated;
+if you are using Maven to install the connector, you must add `<classifier>jar-with-dependencies</classifier>` to your `pom.xml`.
+Also removed the permissions for MongoDB connector.
+https://github.com/hazelcast/hazelcast/pull/25744[#25744], https://github.com/hazelcast/hazelcast/pull/25701[#25701]
+* Changed the return values for `getMethodName()` of the following message task classes:
+** `MapFetchKeysMessageTask`: `iterator` -> `SecurityInterceptorConstants.ITERATOR_FETCH_KEYS`
+** `MapFetchWithQueryMessageTask.`: `iterator` -> `SecurityInterceptorConstants.ITERATOR_FETCH_WITH_QUERY`
+** `MapProjectionWithPredicateMessageTask`: `projectWithPredicate` -> `SecurityInterceptorConstants.PROJECT`
+** `ScheduledExecutorGetAllScheduledMessageTask`: `getAllScheduled` -> `SecurityInterceptorConstants.GET_ALL_SCHEDULED_FUTURES`
+** `ScheduledExecutorSubmitToPartitionMessageTask`: `submitToPartition` -> `SecurityInterceptorConstants.SCHEDULE_ON_PARTITION`
+** `ScheduledExecutorSubmitToTargetMessageTask`: `submitToAddress` -> `SecurityInterceptorConstants.SCHEDULE_ON_MEMBER`
+** `ScheduledExecutorTaskGetResultFromPartitionMessageTask`: `getResultTimeout` -> `SecurityInterceptorConstants.GET_RESULT`
+** `ScheduledExecutorTaskGetResultFromTargetMessageTask`: `getResultTimeout` -> `SecurityInterceptorConstants.GET_RESULT`
+** `SetGetAllMessageTask`: `iterator` -> `SecurityInterceptorConstants.GET_ALL`
+** `TransactionalListAddMessageTask`: `set` -> `SecurityInterceptorConstants.ADD`
+** `TransactionalMultiMapRemoveEntryMessageTask`: `removeEntry` -> `SecurityInterceptorConstants.REMOVE`
++
+https://github.com/hazelcast/hazelcast/pull/25020[#25020]
+* Renamed the service port for Hazelcast clusters deployed in Kubernetes environments as `hazelcast`.
+Previously, the name was `hazelcast-service-port` causing the member auto-discovery (for embedded deployments) to fail.
```

Comment:
The previous name, `hazelcast-service-port`, caused member auto-discovery for embedded deployments to fail.